packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -246,7 +259,7 @@ export class PluginLedgerConnectorQuorum
           `${fnTag} Cannot create an instance of the contract because the contractName and the contractName of the JSON doesn't match`,
         );
       }
-      const contractStr = await keychainPlugin.get(contractName);
+      const contractStr: string = await keychainPlugin.get(contractName);
```

Comment:
@travis-payne Do we still need the `: string` type declaration here? I think the keychain now always returns `string` anyway.

Reply:
Removed